### PR TITLE
Apply rule fix

### PR DESCRIPTION
**Pull request by MichaelBergquistSuarez**
Enable usage of `@apply` funcitonality

Associated pull requests:
https://github.com/viskan/streamline/pull/499
https://github.com/viskan/theme/pull/40
https://github.com/viskan/preprocessor/pull/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "estore-preprocessor",
-  "version": "3.1.4",
+  "version": "3.2.0",
```
Comment (gillstrom) [inline package.json]: Should be in a separate commit, will be weird if another PR is accepted and published to npm. :)

Comment (MichaelBergquistSuarez) [inline package.json]: Sure!:) Done

**Comment by alonalon** [comment]
Good, seems to work like expected. you forgot to add a test. but I do a new PR with test
